src/components/ReadMore/index.jsx

```diff
@@ -18,7 +18,7 @@ const ReadMore = ({ children, postId, picCap = false, readMore = true }) => {
     : text;
 
   return (
-    <>
+    <div style={{zIndex:1,backgroundColor:'white',opacity:0.7}}>
```

Comment:
please avoid inline styling as it reduces readability. You may add a class to this div with suitable name

Reply:
You may also want to check out the other occurrences of this component and see if they're mis-affected.